### ISSUE TITLE
Irish to British Conversion

### ISSUE THREAD

**Issue by BobHarper1** [ideal-postcodes/ospoint#1]
Hello, I have quite an odd use case in that I need to convert from Irish to British **Grid References**. Is it possible using OSPoint?

**Comment by cblanc** [comment]
Hi Bob, it doesn't provide the ability to translate Irish Grid Reference to British Grid Reference

My understanding is that these are just cartesian coordinates. So it may just be a matter of translating the X and Y values by the relative difference between the two origins. The downside (if it matters to you) is that your eastings will be negative for Irish locations.

However this is just a hunch, it's probably more complicated than that. It may be a good idea to reach out to a specialist who knows how to do this.

**Comment by BobHarper1** [comment]
Thanks Chris, I found a way to convert (indirectly) using http://www.howtocreate.co.uk/php/gridref.php (having to go Irish Grid Ref > WGS84 > British Grid Ref). I had assumed it was possible to translate as you describe it there, but as the link also explains, I think it is more complicated. 

As I say, my use case is a bit odd in that I needed to translate from Irish to British so that a third party program can then translate them to WGS84 (but it won't accept WGS84 in the first instance). Unusual, but we use Irish Grid system in Northern Ireland in most cases, so that's what's necessary.

Anyway, OSPoint is a great and simple module and I use it in plenty of other cases, so thanks for including the Irish system.
